workspaces/resource-optimization/plugins/resource-optimization-common/src/generated/apis/DefaultApi.client.ts

```diff
@@ -112,8 +112,14 @@ export class DefaultApiClient {
         endDate?: string;
         offset?: number;
         limit?: number;
-        orderBy?: string;
-        orderHow?: string;
+        orderBy?:
+          | 'cluster'
+          | 'project'
+          | 'workload_type'
+          | 'workload'
+          | 'container'
+          | 'last_reported';
+        orderHow?: 'asc' | 'desc';
```

Comment:
There is issue because of this line https://github.com/jkilzi/backstage-community-plugins/blob/18fdf73d13948543493e7ad4d160cc37b28c7215/workspaces/resource-optimization/plugins/resource-optimization-common/src/generated/apis/DefaultApi.client.ts#L74 
it is expecting `recommendationId` and we are passing same from frontend but it is converting it to `recommendation_id`. Please refer the screenshot for reference 
![image](https://github.com/user-attachments/assets/d04f4479-5945-4018-9aac-dffa56509aea)

we had faced the similar issue before fixed it but I am not sure if something got changed in recent commits. @jkilzi can you please check and fix this? Thanks!

Reply:
OK, I see what's the issue now.

Again, we need to allow the function that handles the case conversion to make an axception for the `recommendationId` path parameter so it doesn't convert it to `recommendation_id`.

Reply:
Addressed in https://github.com/jkilzi/backstage-community-plugins/commit/0ac0c4e0b37b27394a1330cba949d1321922b3e2

https://github.com/user-attachments/assets/caf52d1e-17cc-45d5-bad0-1735bbe612e1